docker/DOCKER.md

```diff
@@ -77,56 +77,68 @@ For guidance on installation, development, deployment, and administration, see o
 
-```bash
-docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:latest start
-```
-
-For just getting started with a development server running in memory, you can pass the container a basic initialization to set the user and password as root and enable logging.
+For just getting started with a development server running in memory, you can start the container with basic initialization arguments to create a root user with "root" as username and password and enable debug logging.
 
 ```bash
-docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:latest start --log trace --user root --pass root memory
+docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:latest start --log debug --user root --pass root memory
 ``` 
 
-Access to the surrealdb CLI with:
+You can access the server using the same SurrealDB CLI provided in the image by using the `sql` command:
 
 ```bash
 docker exec -it <container_name> /surreal sql -e http://localhost:8000 -u root -p root --ns test --db test --pretty
 ```
 
 <h2><img height="20" src="https://github.com/surrealdb/surrealdb/blob/main/img/gettingstarted.svg?raw=true">&nbsp;&nbsp;Run using Docker Compose</h2>
 
+The Docker image can be used with the `docker compose` command.
+
+Here is an example of a basic `docker-compose.yml` file.
+
+```yaml
+services:
+  surrealdb:
+    command: start 
+    image: surrealdb/surrealdb:v2
+    ports:
+      - 8000:8000
+    environment:
+      - SURREAL_LOG=debug
+      - SURREAL_USER=root
+      - SURREAL_PASS=root
+```
+
+Most of the configuration of SurrealDB can be done through [environment variables](https://surrealdb.com/docs/surrealdb/cli/env).
 
-The Docker image can be used with the `docker-compose` tool.
-You can execute the following command to view the help of the `start` command to know the supported _environment variables_:
+You can find a comprehensive list of all the available environment variables in the help message of the `start` subcommand:
 
 ```shell
 docker run --rm surrealdb/surrealdb:latest start --help
 ```
 
-Here is an example of `docker-compose.yml` file.
+SurrealDB can be executed as a non-root user for added security. This ensures that exploiting certain vulnerabilities in the SurrealDB process does not immediately result in privileged access to the container. When doing this, ensure that any files required by SurrealDB are mounted to the container in a volume and that are accessible to that non-root user through their ownership and permissions.
 
-```yaml
-version: '3'
+Here is an example of running the container with a persistent volume as a non-root user with Docker Compose:
 
+```yaml
 services:
   surrealdb:
-    env_file:
-      - .env
-    command: start 
-    image: surrealdb/surrealdb:latest
+    image: surrealdb/surrealdb:v2
```

Comment:
```suggestion
    image: surrealdb/surrealdb:latest
    pull_policy: always
```
?

Reply:
Done in [714219f](https://github.com/surrealdb/surrealdb/pull/5190/commits/714219fa20acd746359c663f98c6623ab0bce297).